aldryn_forms/boilerplates/bootstrap3/templates/aldryn_forms/fields/multipleselectfield.html

```diff
@@ -0,0 +1,31 @@
+{% load aldryn_forms_tags i18n %}
+
+{% if field %}
+    <div class="form-group{% if field.field.required %} form-required{% endif %}{% if field.errors %} has-error{% endif %}">
+        {% block form_field_label %}
+            <label for="{{ field.auto_id }}">{{ field.label }}{% if field.field.required %}<abbr title="{% trans "Required field" %}">*</abbr>{% endif %}</label>
+        {% endblock form_field_label %}
+        {% block form_field %}
+            <select multiple class="form-control" name="{{ field.name }}" id="{{ field.auto_id }}">
+            {% comment %}
+                NOTE: Need to give the possibility to enable/disable te search field in the select dropdown.
+                e.g. data-minimum-results-for-search="-1"
+            {% endcomment %}
+                {% for choice in field.field.choices %}
+                    <option>
+                        {{ choice.1 }}
+                    </option>
+                {% endfor %}
+            </select>
+        {% endblock form_field %}
+        {% block form_field_errors %}
+            {% if field.errors %}
+                {% for error in field.errors %}<span class="text-danger">{{ error }}</span>{% endfor %}
```

Comment:
Guidelines says you have to start code after `{% if %}`, `{% forloop %}`, `{% block %}` with new line and indent.
Please check if it's right for all templates, and correct if it's not harm readability.